_includes/releases/v22.2/v22.2.2.md

```diff
@@ -0,0 +1,69 @@
+## v22.2.2
+
+Release Date: January 4, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-2-sql-language-changes">SQL language changes</h3>
+
+- Fixed a bug where CockroachDB could crash if a user creates a [user-defined function (UDF)](../v22.2/user-defined-functions.html) whose function signature includes an implicit record type which has a column using a user-defined enum type. [#94241][#94241]
+- Added the `log_timezone` [session variable](../v22.2/set-vars.html), which is read only and always UTC. [#94346][#94346]
+- Implemented the `pg_timezone_names` pg_catalog table, which lists all supported timezones. [#94346][#94346]
+
+<h3 id="v22-2-2-db-console-changes">DB Console changes</h3>
+
+- Updated metric graph tooltip styling to prevent content collapse. [#93928][#93928]
+- Updated UI to show correct login information in the top right corner for secure clusters, and fixed documentation links to correctly reference the current cluster version as necessary. [#94067][#94067]
+
+<h3 id="v22-2-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where the `session_id` session variable](../v22.2/set-vars.html) would not be properly set if used from a subquery, [#93856][#93856]
+- Fix a bug that would result in incomplete backups when non-default, non-public resource limiting settings (`kv.bulk_sst.max_request_time` or `admission.elastic_cpu.enabled`) were enabled. [#93728][#93728]
+- Updated the volatility of the `hmac`, `digest`, and `crypt` [built-in functions](../v22.2/functions-and-operators.html) to be immutable. [#93924][#93924]
+- Server logs will now correctly `fsync` at every `syncInterval`. [#93995][#93995]
+- The `stxnamespace`, `stxkind` and `stxstattarget` columns are now defined in `pg_statistics_ext`. [#94009][#94009]
+- The [`CREATE ROLE`](../v22.2/create-role.html), [`DROP ROLE`](../v22.2/drop-role.html), [`GRANT`](../v22.2/grant.html), and [`REVOKE`](../v22.2/revoke.html) statements no longer work when the transaction is in read-only mode. [#94103][#94103]
+- Fixed a bug where CockroachDB could crash in rare circumstances when evaluating lookup and index joins. The bug has been present since the 22.2.0 release. [#94100][#94100]
+- When experimental MVCC range tombstones are enabled (they are disabled by default), a bulk ingestion (e.g. an [`IMPORT`](../v22.2/import.html)) could fail to take a committed-but-unresolved write intent into account during conflict checks when written above an MVCC range tombstone. It was therefore possible in very rare circumstances for the ingestion to write a value below the timestamp of the committed intent, causing the ingested value to disappear. [#94006][#94006]
```

Comment:
These present tense "when blah blah" couple of notes could be reformatted to be more style consistent. Presumably should be something more like "Fixed a bug where, when blah blah"?